docs/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Aiven
+meta_desc: The Aiven provider for Pulumi can be used to provision any of the cloud resources available in Aiven.
+layout: package
+---
```

Comment:
bummer that this page is empty; our hand rolled docs page had a basic example: [registry/themes/default/content/registry/packages/aiven/_index.md at master · pulumi/registry](https://github.com/pulumi/registry/blob/master/themes/default/content/registry/packages/aiven/_index.md)

Reply:
This is on purpose. We are thinking of removing this page altogether and/or moving the `installation-configuration` content here. Having two of these doesn't really add anything and complicates autogeneration.

Reply:
But I agree that we can add maybe a little friendly boilerplate here for now.